### ISSUE TITLE
<T/> adds unwanted spaces

### ISSUE THREAD

**Issue by bfontaine** [HenryLie/svelte-i18n-lingui#13]
Hello,
When using `<T/>`  the resulting text has unwanted spaces:

```html
<T msg="(#)"><a href="#">foo</a></T>
```

Expected result:
```html
(<a href="#">foo</a>)
```

Actual result:
```html
( <a href="#">foo</a> )
```

I think this is because of the newlines here: https://github.com/HenryLie/svelte-i18n-lingui/blob/3380b8c02a9b6862440fc0152b509f682f0bf6e0/src/lib/T.svelte#L65-L75

**Comment by HenryLie** [comment]
Thanks for posting the issue! You're right, those lines are the cause of the issue. It seems I've made a fix in the next branch but haven't backported it to the stable branch just yet:

https://github.com/HenryLie/svelte-i18n-lingui/blob/40795b13d88d5a88434e2510dba3082117f737f4/src/lib/T.svelte#L65-L68

Let me make the fix and release a new version 😄

**Comment by HenryLie** [comment]
@bfontaine this issue is now fixed in the latest version (v0.1.8). 
Could you verify that it works as expected and close this issue?

**Comment by bfontaine** [comment]
Thank you! I confirm it now works as expected.
